plugins/reach/plugins/OpenCalais/batch/kReachVendorTaskOpenCalaisProcessorEngine.php

```diff
@@ -59,7 +63,20 @@ function handleTask(KalturaEntryVendorTask $vendorTask)
         KBatchBase::impersonate($vendorTask->partnerId);
 
         try {
-            $mappingProfileId = $this->getMappingMetadataProfileId();
+            // If entry has PreventAutomatedCategorization set to True --> no tags should be created for the entry.
+            $autoCategorizationMetadataProfileId = $this->getMetaDataProfileId(self::SHOW_DATA_EXTRACTION_WORKFLOW_SYSTEM_NAME);
+            $preventAutoCategorizationMetadata = $this->retrieveMetadataObjectsByMetadataProfileAndObjectId($autoCategorizationMetadataProfileId, KalturaMetadataObjectType::ENTRY ,$vendorTask->entryId);
+            if ($preventAutoCategorizationMetadata->totalCount)
```

Comment:
here as well i would check the object is not null before accessing its fields

Reply:
@inbal-ben-david  see above.